src/Xamarin.Android.Build.Tasks/Tasks/GenerateResourceDesigner.cs

```diff
@@ -70,11 +72,14 @@ public override bool RunTask ()
 
 			// Create our capitalization maps so we can support mixed case resources
 			foreach (var item in Resources) {
-				if (!item.ItemSpec.StartsWith (ResourceDirectory))
+				var path = Path.GetFullPath (item.ItemSpec);
+				if (!path.StartsWith (ResourceDirectory))
```

Comment:
This `.StartsWith()` should be `path.StartsWith (ResourceDirectory, StringComparison.OrdinalIgnoreCase)`…